XIVSlothCombo/Combos/PvE/AST.cs

```diff
@@ -30,6 +30,7 @@ internal const uint
 
             //Cards
             AstralDraw = 37017,
+            UmbralDraw = 37018,
```

Comment:
Similar to comment above, not sure how these AST changes are relevant to the focus of this NIN PR.